Formula/amdatu-bootstrap.rb

```diff
@@ -19,9 +19,16 @@ class AmdatuBootstrap < Formula
   depends_on "openjdk@8"
 
   def install
+    env = Language::Java.java_home_env("1.8")
+    if OS.linux?
+      # Add java to PATH to fix: amdatu-bootstrap: line 35: java: command not found
+      env["PATH"] = "$JAVA_HOME/bin:$PATH"
+      # Use bash instead of dash to fix: amdatu-bootstrap: 34: [: --info: unexpected operator
+      inreplace "amdatu-bootstrap", %r{^#!/bin/sh$}, "#!/bin/bash"
+    end
```

Comment:
We will probably lose `all` bottle due to this.

Could preserve by applying changes to both OS:
- Switching `/bin/sh --> /bin/bash` should be fine on macOS too.
- Not sure if we want to shadow the default `/usr/bin/java` which automatically looks at `JAVA_HOME`. Though, I think, `$JAVA_HOME/bin/java` should work.

Reply:
`write_jar_script` uses `$JAVA_HOME/bin/java` so I'd remove the Linux conditional.